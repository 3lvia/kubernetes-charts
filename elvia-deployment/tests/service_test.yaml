suite: test service
templates:
  - service.yaml
tests:
  - it: should work
    set:
      image.tag: latest
      name: demo-api
      namespace: core
      microserviceType: webapi
      environment: dev
      replicaCount: 1
      resources:
        requests:
          cpu: 100m
          memory: 128Mi
        limits:
          cpu: 400m
          memory: 500Mi
      ingress:
        subdomain: core
        path: "/demo-api"
    asserts:
      - matchSnapshot: {}
