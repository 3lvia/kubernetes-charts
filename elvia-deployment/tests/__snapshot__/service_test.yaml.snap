should work:
  1: |
    apiVersion: v1
    kind: Service
    metadata:
      labels:
        app: demo-api
      name: demo-api
      namespace: core
    spec:
      ports:
        - name: demo-apiport
          port: 2112
      selector:
        app: demo-api
